packages/rabbitmq/src/rabbitmq.interfaces.ts

```diff
@@ -45,8 +45,16 @@ export interface MessageHandlerOptions {
   queue?: string;
   queueOptions?: QueueOptions;
   errorBehavior?: MessageHandlerErrorBehavior;
+  errorCallbacks?: IMessageErrorCallback[];
+  allowNonJsonMessages?: boolean;
```

Comment:
I like the intent for `allowNonJsonMessages` but I think that it doesn't belong. One of the follow up things I'd like to do with the RabbitMQ library is provide pluggable serialization/deserialization of messages instead of assuming JSON on the users's behalf. For now I'd rather just let the JSON parsing throw an error and leave it to the errorCallback to decide what to do with the error

Reply:
All right, but we should do something about empty messages. At this point, sending empty (i.e. empty byte array) to the queue will result in an JSON parsing error.